packaging/conda/meta.yaml

```diff
@@ -1,6 +1,8 @@
+{% set version = os.environ.get('KHIOPS_VERSION') %}
+
 package:
   name: khiops-binaries
-  version: {{ os.environ.get('KHIOPS_VERSION') }}
+  version: {{ version.replace("-", ".") }}
```

Comment:
With the modification in the comment above, we can write directly:
```
version: {{ version }}
```

for all output packages.

Reply:
done